Frameworks/include/CFCppBase.h

```diff
@@ -0,0 +1,90 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
```

Comment:
Yes, it is a super low priority, and will likely result in a ton of conflicts trying to merge multi-way.  (We have 3 active branches + develop across all of winobjc).  Let us update what we can as we go about our daily business.

---
In reply to: [83494841](https://github.com/Microsoft/WinObjC/pull/1153#discussion_r83494841) [](ancestors = 83494841,83494095)